shell/src/main/java/alluxio/cli/fs/command/ChmodCommand.java

```diff
@@ -103,12 +103,12 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
 
   @Override
   public String getUsage() {
-    return "chmod [-R] <mode> <path>";
+    return "chmod [-R(--recursive)] <mode> <path>";
```

Comment:
can you check other classes in the CLI to see what's the better way to say -R or --recursive?
I am asking because you use two formats 
[-R --recursive]
[-R(--recursive)]

Reply:
I changed to -R/--recursive, which I think is more user-friendly